engine/gamesys/src/gamesys/test/test_gamesys.cpp

```diff
@@ -1438,7 +1438,7 @@ TEST_F(GamepadConnectedTest, TestGamepadConnectedInputEvent)
     dmGameSystem::FinalizeScriptLibs(scriptlibcontext);
 }
 
-TEST_F(Sleeping2DCollisionObjectTest, WakingCollisionObjectTest)
+TEST_F(CollisionObject2DTest, WakingCollisionObjectTest)

```

Comment:
As shown in the previous physics PR, we also need to add a "compile time" test in `test_physics.cpp`.
This is needed to make sure we catch missing features early on.
In this case, there is no change to `physics_2d_null.cpp`, which is required.

Reply:
@JCash , please elaborate a little here. wdym by 'compile time' test ? A test that will involve calling  dmPhysics::SetBullet2d/IsBullet2D() that will fail if the feature is not present ?

I went through physics_2d_null.cpp. I noticed that it contains dummy implementations of functionality typically present in physics_2d.cpp. 

Could you briefly explain how this works ?

Reply:
Nevermind. We solved this.